sherpa.spec

```diff
@@ -5,6 +5,10 @@
 Source: git+https://github.com/%github_user/%{n}.git?obj=%{branch}/%{tag}&export=%{n}-%{realversion}&output=/%{n}-%{realversion}-%{tag}.tgz
 Requires: hepmc lhapdf blackhat sqlite fastjet openssl
 
+%if "%(case %cmsplatf in (*amd64*) echo true ;; (*) echo false ;; esac)" == "true"
```

Comment:
Ah, I see. Our triplets aren't that good, missing "linux", "darwin", "freebsd", etc. We don't have a type of OS.